docs/conf.py

```diff
@@ -102,11 +120,12 @@
 # -- Options for LaTeX output ------------------------------------------------
 
 # Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title,
-#  author, documentclass [howto, manual, or own class]).
+# (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [('index', project + '.tex', project + u' Documentation',
                     author, 'manual')]
 
+latex_logo = '_static/astropy_logo.pdf'
+
```

Comment:
`latex_logo` should be removed IMHO, the `_static` folder is not in git.

Reply:
Yes, makes sense. Done.